include/grpc++/security/server_credentials.h

```diff
@@ -39,6 +39,7 @@
 
 #include <grpc++/security/auth_metadata_processor.h>
 #include <grpc++/support/config.h>
+#include <grpc/grpc_security.h>
```

Comment:
The best I've got is maybe introducing an enum-only header for the security bits.